docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -654,3 +654,12 @@ By default all blocks can be converted to a reusable block. If supports reusable
 reusable: false,
 ```
 
+- `pasteTextInline` (default `false`): If a single line of text is pasted into the block editor, resulting
+in this block being created, setting `pasteTextInline` to `true` will instead extract the text content of
+what is being pasted, and insert that into the currently selected block, instead.
```

Comment:
> If a single line of text is pasted into the block editor, resulting in this block being created [...]

This is not entirely true. It will do this for any block created when the pasted plain text had no line breaks. Additionally, this option is quite difficult to understand when to apply it correctly. Maybe `containsOnlyRichText` might be easier to understand?